Fix failed test on TestParameter

```
--- FAIL: TestParameter (0.00s)
    --- FAIL: TestParameter/normal_input (0.00s)
        /Users/tchiunam/repo/axolgo-lib/types/types_test.go:53: 
            	Error Trace:	/Users/tchiunam/repo/axolgo-lib/types/types_test.go:53
            	Error:      	Not equal: 
            	            	expected: string("hammer")
            	            	actual  : *string((*string)(0xc000099060))
            	            	
            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -1 +1 @@
            	            	-hammer
            	            	+<*string Value>
            	Test:       	TestParameter/normal_input
            	Messages:   	Expected name hammer, got %!s(*string=0xc000099060)
        /Users/tchiunam/repo/axolgo-lib/types/types_test.go:54: 
            	Error Trace:	/Users/tchiunam/repo/axolgo-lib/types/types_test.go:54
            	Error:      	Not equal: 
            	            	expected: string("nail")
            	            	actual  : *string((*string)(0xc000099070))
            	            	
            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -1 +1 @@
            	            	-nail
            	            	+<*string Value>
            	Test:       	TestParameter/normal_input
            	Messages:   	Expected value nail, got %!s(*string=0xc000099070)
FAIL
FAIL	github.com/tchiunam/axolgo-lib/types	0.397s
FAIL
```

--- a/types/types_test.go
+++ b/types/types_test.go
@@ -50,8 +50,8 @@ func TestParameter(t *testing.T) {
 			// initialize the parameter
 			param := Parameter{Name: tc.name, Value: tc.value}
 			// check the name and value
-			assert.Equal(t, *tc.name, param.Name, "Expected name %s, got %s", *tc.name, param.Name)
-			assert.Equal(t, *tc.value, param.Value, "Expected value %s, got %s", *tc.value, param.Value)
+			assert.Equal(t, *tc.name, *param.Name, "Expected name %s, got %s", *tc.name, *param.Name)
+			assert.Equal(t, *tc.value, *param.Value, "Expected value %s, got %s", *tc.value, *param.Value)
 		})
 	}
 }

